### ISSUE TITLE
Not creating Folder

### ISSUE THREAD

**Issue by RealMedman** [CoffeetimeSolutions/SharePoint-ListLibraryRelation#1]
Hi
What a wonderfull script. :) 
I have one problem, the script does not create the folders, if I create them manualy after the creation it works. i'm on Sharpoint Online.
Your guide is a little bit hard to follow in step 6

`~site/SiteAssets/js/ListFormCSR.js
?Do the same "As above?" for the new and edit form of the list.

Step 6
**Onto this page now**? also insert the document library webpart. Click “Edit web part” and add another “JSLink”:`

**Comment by andybandy89** [comment]
Hi RealMedman,
yes, you are right. I'll have to improve the description at the end. Indeed it is confusing. 

Just tested the script in SharePoint Online (it was developed for 2013) and it works but you will have to change the customs list experience to "Classic Experience". Otherwise you are not able to link the JSLink to the lists Display-, Edit- and NewForm.

If you have changed to the classic experience click on new item. Now you can click onto the gears symbol and there you will find "Edit page". You will see that this is just a webpart page and you are able to set the JSLink: 

> ~site/SiteAssets/js/jquery-3.1.0.min.js | ~site/SiteAssets/js/ListFormCSR.js

Do exactly the same on the DisplayForm (if you view the item) and on the EditForm (if you edit the item). 
On the display form there is one additional step necessary:

Also add there the webpart of the document library. And to the document library webpart add this JSLink:

> ~site/SiteAssets/js/jquery-3.1.0.min.js | ~site/SiteAssets/js/DoclibFormCSR.js

Hopefully I explained it better this time :-)
